### PR TITLE
Add organization

### LINKED ISSUE #37: Option to add new organizations is not working anymore
After updating DJango version the option to add a new organization during a user registration is not available anymore. We have to fix it or redesign it, because previous solution was quite dirty.

### PR DESCRIPTION
**Pull request by ref-humbold**
Fix #37 #51

### CODE REVIEW

**Comment by Bozydarek** [comment]
What else is missing to finish this PR?

**Comment by ref-humbold** [comment]
> What else is missing to finish this PR?

Real integration via API to add new organization and choose it. Currently there is only a placeholder that adds only "KSI" organizations.

File: app/src/conferences/templates/conferences/register.html [hunk 1]
```diff
@@ -1,8 +1,33 @@
...
 {% load materializecss %}
 
 {% block content %}
+<!-- Modal Structure -->
+<div id="add_org" class="modal">
+  <div class="modal-content">
+    <h4>Add organization</h4>
+    <div class="progress">
+      <div id="progress_type" class=""></div>
+    </div>
```
[the remark this answers is not in the document]
Comment (JakubSzczerbinski) [reply app/src/conferences/templates/conferences/register.html]: If network is slow we want to use some kind of progress bar to show we made a request to server. Enable throttling in chrome dev tools to see the effect.

**Comment by JakubSzczerbinski** [comment]
@Bozydarek 
> Also we need to sort list of organization by acceptance and name.

It should be done on server side @ref-humbold. There should be GET endpoint for organization list.
From my POV, flow should be:
- Add org via api
- Get organization list via api
- Update select with organization list
> Plus it would be nice to select automatically organization added by user.

Done.
> Moreover why we limit organization list only to accepted and defined by user?

Are you talking about frontend or backend?

**Comment by ref-humbold** [comment]
I created endpoint `api/v1/users/organizations/` for adding (POST) and retrieving (GET) organizations. GET returns list of all organizations.

**Comment by ref-humbold** [comment]
First goes PR #63

**Pull request merged**